docs/proposals/ThumbnailGallery/README.md

```diff
@@ -0,0 +1,61 @@
+# Component Name
+
+The `ThumbnailGallery` component will accept a list of attachments for images
+and display them as thumbnails. This would also have an option to show the
+images in a LightBox component on click of those images.
+
+## Accessibility
+
+Assistive technology should be able to see the images and know the image name
+based on the `name` property of each attachment.
```

Comment:
additional considerations:

- How will users navigate the gallery with a keyboard
- How will "count of images left" blocks announce themselves to screen-readers

@chriscarline any other thoughts? [Checklist](https://jobber.atlassian.net/wiki/spaces/DES/pages/631604349/Accessibility+Checklist) for reference

Reply:
Navigation between the thumbnails can be with `Tab` and `Shift+tab`(for backwards navigation). Order would be left to right.

I'm not as familiar with the standards for naming elements for screen readers but something like `[# of extra attachments] more attachments, button` if the reader is able to know the number of attachments. If not, maybe something like `More attachments, button` would work.